lib/services/ngsi/ngsiService.js

```diff
@@ -106,6 +124,73 @@ function generateNGSIOperationHandler(operationName, entityName, typeInformation
 }
 
 /**
+ * Generate an operation handler for NGSIv2-based operations (query and update). The handler takes care of identifiying
+ * the errors and calling the appropriate callback with a success or a failure depending on how the operation ended.
+ *
+ * Most of the parameters are passed for debugging purposes mainly.
+ *
+ * @param {String} operationName        Name of the NGSI operation being performed.
+ * @param {String} entityName           Name of the entity that was the target of the operation.
+ * @param {Object} typeInformation      Information about the device the entity represents.
+ * @param {String} token                Security token used to access the entity.
+ * @param {Object} options              Object holding all the information about the HTTP request.
+
+ * @return {Function}                  The generated handler.
```

Comment:
Missaligned (just one whitespace)?

Reply:
NTC.

It follows the same pattern that the rest of functions. In fact, it is copied from generateNGSIOperationHandler function.